iree/hal/interpreter/CMakeLists.txt

```diff
@@ -82,8 +82,6 @@ iree_cc_library(
     bytecode_kernels
   HDRS
     "bytecode_kernels.h"
-    "bytecode_kernels_generic.h"
-    "bytecode_kernels_ruy.h"
```

Comment:
I think these are in textual headers, which are currently just added to the HDRS in CMake.

Reply:
Should we avoid making this change? It doesn't cause any new build or test failures, but maybe there's something that isn't covered?

Reply:
I am not sure. After reading the [bazel docs](https://docs.bazel.build/versions/2.0.0/be/c-cpp.html#cc_library.textual_hdrs), I think we might be need to add the textual headers as `PUBLIC` [target_sources](https://cmake.org/cmake/help/latest/command/target_sources.html) instead?

Reply:
I could error out if the textual_hdrs kwarg is present until we figure out how to support it

Reply:
How about reverting in this PR, then figuring out separately how we want to handle `textual_hdrs`?

Reply:
The idea with strict mode is that we can run it safely, so I'll add a failure on textual_hdrs in #592 and then rerun it.

I wonder if we should actually remove all **kwargs from the rule defs so that any new parameter we haven't thought of errors out

Reply:
I made the change in #592 and reran everything. Had to add some additional magic incantations to avoid failures in the new strings module